lib/eventmodel.js

```diff
@@ -257,7 +257,9 @@ EventModel.prototype._removeItemContext = function(context) {
 
 EventModel.prototype._addBinding = function(binding) {
   var bindings = this.bindings || (this.bindings = new BindingsMap());
-  binding.eventModels || (binding.eventModels = new EventModelsMap());
+  if (bindings.eventModel == null) {
```

Comment:
This should actually be:
```js
if (binding.eventModels == null) {
```

Issue was caught by internal tests when testing derby-2 branch. I'll try and write a more minimal test in the Derby tests.